relay-config/src/redis.rs

```diff
@@ -70,6 +70,18 @@ enum RedisConfigFromFile {
         options: PartialRedisConfigOptions,
     },
 
+    /// Connect to a set of Redis instances for multiple writes.
+    MultiWrite {
+        /// List of `redis://` urls to use in multi write mode.
+        ///
+        /// This can also be a single node.
+        nodes: Vec<String>,
```

Comment:
I think this should mirror the connection structure and allow for a recursive nesting of connections. 

Basically something like `MultiWrite(Vec<RedisConfigFromFiel>)` here. That allows proper mixing of cluster/non-cluster instances and is a logical extension to the current configs.

Reply:
Yeah, could also be an option. I initially built the config without such recursion in mind, but from our chat we planned on doing that, so I will do it now, thanks!